lib/span-data.js

```diff
@@ -82,6 +92,23 @@ SpanData.prototype.close = function() {
 };
 
 /**
+ * Trace API expects stack frames to be a JSON string with the following
+ * structure:
+ * STACK_TRACE := { "stack_frame" : [ FRAMES ] }
+ * FRAMES := { "class_name" : CLASS_NAME, "file_name" : FILE_NAME,
+ *             "line_number" : LINE_NUMBER, "method_name" : METHOD_NAME }*
+ * @constructor @private
+ */
+function StackFrame(className, methodName, fileName, lineNumber, columnNumber) {
+  if (className)   { this.class_name = className;  }
```

Comment:
My thinking was that I want to leave the property undefined, as opposed to being set to null, if null is passed in; this reduces the data sent on the network. For example, className will always be null in our case. Upon reflection, this does the wrong thing for lineNumber or columnNumber when they are 0. I'll try to fix this.